<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1311</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon final.ico</normaloff>icon final.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/newPrefix/Light-blue-tech-background-vector-02.jpg);</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonFollowStyle</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="grBox1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1291</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="butOk">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>75</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QLabel" name="lCon">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>0</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>СОЕДИНЕНИЕ С VK.COM</string>
     </property>
    </widget>
    <widget class="QLabel" name="lLog">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>ЛОГИН</string>
     </property>
    </widget>
    <widget class="QLabel" name="lPass">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>75</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>ПАРОЛЬ</string>
     </property>
    </widget>
    <widget class="QLabel" name="lStat">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>75</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Статус: нет соединения</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edLog">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>40</y>
       <width>211</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="edPass">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>75</y>
       <width>211</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lImg">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>111</width>
       <height>105</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>111111.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lLog_2">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>30</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>История операций</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="history">
     <property name="geometry">
      <rect>
       <x>960</x>
       <y>60</y>
       <width>321</width>
       <height>111</height>
      </rect>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grBox2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>611</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="butIDOk1">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>75</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QLabel" name="lUser">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>0</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>ПОЛЬЗОВАТЕЛЬ</string>
     </property>
    </widget>
    <widget class="QLabel" name="lID1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>44</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edID1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>191</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="butInfo1">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Информация</string>
     </property>
    </widget>
    <widget class="QPushButton" name="butFrien">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>75</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Список друзей</string>
     </property>
    </widget>
    <widget class="QPushButton" name="butGr">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>110</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Список групп</string>
     </property>
    </widget>
    <widget class="QPushButton" name="butNetw1">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>145</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Построение эго-сети</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bAnalys1">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>180</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Анализ графа</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bPR1">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>40</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>PageRank</string>
     </property>
    </widget>
    <widget class="QPushButton" name="butFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>215</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Файл</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edFile">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>216</y>
       <width>161</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Файл не выбран</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bTriad1">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>75</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Триад. закрытие</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bClust1">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>110</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Коэф. кластеризации</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bDom1">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>145</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Домин. множество</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bGraph1">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>215</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Визуализация графа</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grBox3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>611</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="butIDOk2">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>75</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QLabel" name="lGroup">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>0</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>СООБЩЕСТВО</string>
     </property>
    </widget>
    <widget class="QLabel" name="lID2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>43</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edID2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>191</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="butNetw2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>180</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Построение сети</string>
     </property>
    </widget>
    <widget class="QPushButton" name="butUsers">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>75</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Список участников</string>
     </property>
    </widget>
    <widget class="QPushButton" name="butInfo2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Информация</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bGraph2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>40</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Визуализация графа</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bPR2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>75</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>PageRank</string>
     </property>
    </widget>
    <widget class="QPushButton" name="butLike">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>110</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Статистика(лайки)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="butComm">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>145</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Статистика(коммент)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bTriad2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>110</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Триад. закрытие</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bClust2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>145</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Коэф. кластеризации</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bDom2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>180</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Домин. множество</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bAnalys2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>215</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Анализ графа</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>210</y>
      <width>661</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTextEdit" name="textField">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>641</width>
       <height>471</height>
      </rect>
     </property>
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="bClear">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>495</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Очистить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bExit">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>495</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Выход</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bSave">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>495</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
